### DISCUSSION TITLE
Grouped Updates: add interactive checkbox to dependency table to create specific depedency update individually

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#32427] [Suggest an Idea]
### Tell us more.

I commonly use group presets or custom package rules to group dependency updates to reduce the amount of PRs we get.

However, from time to time the grouped PR contains a single dependency update that includes a breaking change.

It would be great if the dependency dashboard table had a new column that had interactive checkboxes (for GitHub) that allowed you to manually raise grouped updates as a separate PR.

**Comment by rarkins** [comment]
What if instead: the PR contained a checkbox per dependency and you could tick any to exclude it from the group?
